dali/dfu/dfuwu.cpp

```diff
@@ -1680,6 +1680,18 @@ class CDFUfileSpec: public CLinkedDFUWUchild, implements IDFUfileSpec
         queryRoot()->setPropBool("@remoteGroupOverride",set);
     }
 
+    bool getPreserveCompression() const
```

Comment:
The functions for the options are used, but I can't see any use for these 2 (IDFUFileSpec has not been modified)